pkg/catalogsourceconfig/handler.go

```diff
@@ -73,6 +74,9 @@ func (h *catalogsourceconfighandler) Handle(ctx context.Context, in *v2.CatalogS
 	if updateErr := h.client.Update(ctx, out); updateErr != nil {
 		log.Errorf("Failed to update object - %v", updateErr)
 
+		// Error updating the object - report a failed sync.
+		operatorstatus.SendSyncMessage(updateErr)
```

Comment:
Thinking a bit more about this... should we just send the sync message on every error here? i.e. move the send to call sites of `Handle()`.

Reply:
I don't believe we would want to do that.

An [error from the reconcile](https://github.com/operator-framework/operator-marketplace/pull/214/files/0a5750740949bcd69a20cef63aed2bcaa96eca0f..af6316a7c9c8fb71dd36d6b33dfa534b6108617d#diff-f46f13415712030379e27befaa786593R57) call would increment the number of failed syncs - which is more or less what we do today.